Sources/CryptoBoringWrapper/EC/EllipticCurve.swift

```diff
@@ -72,6 +72,14 @@ extension BoringSSLEllipticCurveGroup {
         return try! ArbitraryPrecisionInteger(copying: baseOrder)
     }
 
+    @usableFromInline
+    package var generator: EllipticCurvePoint { get throws {
```

Comment:
Might be worth not squishing this in, `get throws {` can be on its own line.

Reply:
OK, I moved this onto its own line. 

Aside: I have a task to add the swift-format job to this repo soon, which will squash all these inconsistencies away.